boulders/views.py

```diff
@@ -1,3 +1,12 @@
 from django.shortcuts import render
+from .models import Boulder
 
 # Create your views here.
+def boulder(request, pk):
+    boulder = Boulder.objects.get(pk=pk)
```

Comment:
check if the object exists or make use of try except whenever using objects.get() method